From 6b1d976a7860d515dc3b39fbda322be5a833e5ef Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jakub=20=C5=BBurkowski?= <jakub.zurkowski@qt.io>
Date: Mon, 27 Jul 2026 08:29:06 +0200
Subject: [PATCH] manual: register the Squish image

Register the Squish image so the pipelines resolve its name.
---
 gitops/workloads/values-jenkins.yaml | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/gitops/workloads/values-jenkins.yaml b/gitops/workloads/values-jenkins.yaml
index a642eb0..231607a 100644
--- a/gitops/workloads/values-jenkins.yaml
+++ b/gitops/workloads/values-jenkins.yaml
@@ -48,6 +48,8 @@ config:
             image: "horizon-sdv/cloud-ws-images/horizon-asfp"
           horizonAndroidStudio:
             image: "horizon-sdv/cloud-ws-images/horizon-android-studio"
+          horizonSquishForAndroid:
+            image: "horizon-sdv/cloud-ws-images/horizon-squish"
     common:
       serviceAccounts:
         terraformWorkloadsSA:
@@ -524,6 +526,8 @@ controller:
                   value: {{ .Values.config.workloads.cloudWorkstations.workstationPresets.wsImages.horizonASFP.image }}
                 - key: "CLOUD_WS_HORIZON_ANDROID_STUDIO_IMAGE_NAME"
                   value: {{ .Values.config.workloads.cloudWorkstations.workstationPresets.wsImages.horizonAndroidStudio.image }}
+                - key: "CLOUD_WS_HORIZON_SQUISH_FOR_ANDROID_IMAGE_NAME"
+                  value: {{ .Values.config.workloads.cloudWorkstations.workstationPresets.wsImages.horizonSquishForAndroid.image }}
         groovy:
           - script: >
               println("This is groovy !");
-- 
2.43.0

